contracts/suilend/sources/obligation.move

```diff
@@ -169,7 +194,11 @@ module suilend::obligation {
         let mut deposited_value_usd = decimal::from(0);
         let mut allowed_borrow_value_usd = decimal::from(0);
         let mut unhealthy_borrow_value_usd = decimal::from(0);
+        let is_emode = is_emode(obligation);
+        if (is_emode) {
+            obligation.refresh_emode(reserves, is_emode, clock)
```

Comment:
nit: just return immediately

Reply:
assigned it to a variable to avoid borrow-checker error

Reply:
if (is_emode) {
            obligation.refresh_emode(reserves, is_emode, clock)
            return;
}